plugin-build/src/main/kotlin/io/sentry/android/gradle/tasks/SentryUploadProguardMappingsTask.kt

```diff
@@ -30,8 +32,8 @@ abstract class SentryUploadProguardMappingsTask : Exec() {
     val uuidFile: Provider<RegularFile>
         get() = uuidDirectory.file("sentry-debug-meta.properties")
 
-    @get:InputFile
-    abstract val mappingsFile: RegularFileProperty
+    @get:InputFiles
```

Comment:
I think that did the trick to solve what I have tried before and it failed :D, Files instead of File